pydantic_ai_slim/pydantic_ai/toolsets/function.py

```diff
@@ -47,6 +47,7 @@ class FunctionToolset(AbstractToolset[AgentDepsT]):
     docstring_format: DocstringFormat
     require_parameter_descriptions: bool
     schema_generator: type[GenerateJsonSchema]
+    defer_loading: bool
```

Comment:
Need to cover test defaulting here, that defer_loading on FunctionToolset applies to inner tools unless they override this setting.